src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/chain/log.rs

```diff
@@ -0,0 +1,43 @@
+use crate::chain::chain_data::ChainData;
+use crate::chain::chain_event::ChainEvent;
+use crate::hex::{hex_to_int, int_to_hex};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Log {
+    // Log arguments
+    pub data: String,
+    // The block number with the chain.
```

Comment:
Mb _from_ the chain _of_ the deal? Or is it here again that totally confusing `s/deal/chain/g` conversion?

Reply:
yeah, it is the confusing conversion. I was confused by it all the time 😅, but haven't thought it was of my doing.